example/example.cabal

```diff
@@ -0,0 +1,57 @@
+cabal-version: 1.12
```

Comment:
Possibly should be a newer version, although I'm not sure that it matters much.

Reply:
this is what appears in some plutus packages. all this stuff is just from the stack or cabal-install new package template